src/distribution/UnsafeUniformBigIntDistribution.ts

```diff
@@ -31,7 +31,7 @@ export function unsafeUniformBigIntDistribution(from: bigint, to: bigint, rng: R
   while (true) {
```

Comment:
Note: replacing "const MaxAcceptedRandom = FinalNumValues - (FinalNumValues % diff);" by a division does not increase the throughput. It seems to make the situation even worst!